core/src/test/java/org/mskcc/cbio/portal/scripts/TestIntegrationTest.java

```diff
@@ -149,6 +161,23 @@ public void testLoadStudyEs0() {
                 }
             }
             assertEquals(63, countAMP_DEL);
+            //log2CNA
+            geneticProfileStableIds = new ArrayList<String>();
+            geneticProfileStableIds.add("study_es_0_log2CNA");
+            hugoGeneSymbols = new ArrayList<String>(Arrays.asList("ACAP3","AGRN","ATAD3A","ATAD3B","ATAD3C","AURKAIP1","ERCC5"));
+    		cnaProfileData = apiService.getGeneticProfileData(geneticProfileStableIds, hugoGeneSymbols, null, null);
+    		//there is data for 7 genes x 778 samples:
+    		assertEquals(7*778, cnaProfileData.size());
+    		//there are 273 CNA entries that have value between -0.6 and 0.5;
+    		int count0506 = 0;
+            for (Serializable profileEntry: cnaProfileData) {
+                String profileData = ((DBSimpleProfileData)profileEntry).profile_data;
+                double profileDataValue = Double.parseDouble(profileData);
+                if (profileDataValue > -0.6 && profileDataValue <= -0.5) {
```

Comment:
for CNA data we have discrete and continuous values ("log2CNA"). These last ones are used in various plots, such as the mutations vs CNA plot in the study view. 

The if statement here is on purpose, as I am counting the number of values in this loop that are in the range 0.5 to 0.6. I manually verified in the file that this should be 273 values, and this is asserted here in this test (row 180). 

I will add a comment to this block to make this a bit more clear.